tf_quant_finance/examples/jupyter_notebooks/American_Option_Black_Scholes.ipynb

```diff
@@ -1,958 +1,950 @@
kDxwvE2XUGo45Fy7OITQzx2PO/qi9aS3i/7K8oqoJWo+XJ0Q8xLXHiHVEL63L0\n0Vr9MQ6r2OThG5xORNIDaLR6rAO9FG7+Az3t4nDvmOTRTFnyJDSVYznwJrgcgAbjlCex18XgausH\nwN7TQE/ZZ5jSUol7YS1a053RHs+TZBCUJEmSvlMUVeWvBy5z6AvReCA00Iefr84lOsx9x5GaXjN/\nqWrEroj9k0sSI5kybPDg1dBwncLC/bhcN45wmTydtLQMj43/wuk6TheKWUyjj57FD44hOs77S5Fn\nms7z0aVPUFHRa/U8k/UoWRGjvD0sAJz2blqrPsJpF0u5fqFZhA+/D41Gh7mvi8JNb9LX1QpAwshc\nJi18FOXaF1gP/glUF2h0GKc+g60qFOVGqLV1XqXn0k78s0YT89zzaD3UEeZOI4OgJEmS9J3hUlQ+\n2FPB8RKxNBgRbOIXq3OJCHE/k3O5Z4CPq5pwqioaYFlSFBMiB68hrKur5ejRAhRFQaPRMG3aLJKT\nR3hk7Kqqcu74Nc6fEEuXJl8DS1aOIfJLlrKHyrGG02yo3AKAUWvgB9lPkBGW5uVRCQ5rB63VH+Fy\niKXcgPDxhCYsQqPR0NfVRuHmNzH3isO9k0ffzYS8VbiqTmA9+h6i+FKPz7TnsZabUPpE1xhrWyW9\nlXsJGD+BmKefRaP/7sal7+6bSZIkSd8rTpfCaxsv3AyB0WF+/Hx1LqGBPm7vKe/qZ/2VZlyqihZY\nnjKMnPDBZ9+uXq3m+PHDqKqKVqtl+vQ5JCYme2Tsqqpy6nANxWfrAfDzN5K/OpuwCO/Xox2qO8rm\n6p0AmHQ+/HDsGkaEeOa9vy67pYXW6o9RnOLYnsCouwmJzUOj0dDd3siRzX/AOiACYlruDHJnLsNR\ndgjbyY/FA/RGfKasxVKiQb1xYLil6SJ91QcJmjaNYY89icZD50DeqWQQlCRJkr71HE6Ft7aVcqFK\n9LiNj/TnZ6tyCfZ3v5xX3NHHpzXNKIBOA6tSYxgdOngNYXV1JSdPiu4TOp2OmTPziItL9MjYVVXl\n2P5qyi6I+rWAIB/yV2UT8iVL2UNlb+1BdtTsA8BP78sLOU8zPMj7nUwAbAP1tF75K6rHqwfRAAAg\nAElEQVRLzOIFR88gKHo6Go2GzuY6jmz5A3arWObNnDSfrCmLsBfvwn52k3iAwRfjlB9jueBEtYla\nz4Hr5xioPU7InDwiV67+zodAkEFQkiRJ+pazO1y8sfUipTVi+S8pOpCXVuYQ4Gtwe8/5th621Lai\nAnqNhodHxJAeMvjsW0VFGWfPngBAr9cze/YCoqPd9yW+HYqiUrinksqLLQAEhZi4d/VYAoNNHnn+\nV6WqKttr9rL/2mEAAg0BvJj7DHEBMV4d199Y+2ppq9mAqohZvJC4eQRFieNr2uqvcPSzt3DaxS7g\n7HvuJWPCHOznNmMvEjObGp8ADHf/BMt5C9zo1NJ/9Tjm+nOELc4nfOn9d0RnlKEgg6AkSZL0rWW1\nO3ltUwkVdd0AZAwP5YVlY/Azuf96O9XSzY46sbPUqNXwaFosqUGDz76VlRVz/vwZAAwGA3PmLCQq\nyjMdPVwuhUM7K6i+JMYSGu5H/qps/L9kKXsoqKrKpqrtFNaL8BtsDGJt7rNE+3v/EGsAS89l2q9u\nQlXFLF5YwhICIsYB0HS1nBM73sXlFLuAx89ZTmr2VGwn1+EoKwBA4xeCYfyPsZztA0VFVVX6rxzC\n0lRCxAMrCFu42Fuv5hUyCEqSJEnfSmarg1c/LeZKg6gBGzU8lP/3qUlYzXa39xxt6mJvvVg+9tFp\neSItluGBt24kUVWVkpIvKC4+D4DR6ENe3iLCw933Jb4dTqfCgc/Kqa0WXSvCo/zJX5WNr593d6Yq\nqsKGyi2caDwrxmUKZW3us0T4um/FN5TMXeW01/7/7L1neBRXtqj9VmflnEWQEJJAIBBgcjTJGEww\n2MYYnMEJh/HYM2fu9+vc57lzz5zxJOc0DuMENsEmmpyNCQYFhBBISEgop1bq3LW/H8XROXORPAa3\naWHv9x90dbGqulG9WnuvtTYAKqAQ1W8RQZFa257K87l8u+1DVNWLougYPfs++mWOwHHgPTznDwOg\nBEdhHPYM9hNWECCESlvx1zgbiom9bwXh06b77+L8hBRBiUQikdx0tNtc/HltHpfqtHFg2QOieHLh\nECwmQ7ciKIRgb3Uze6q15eMAvY6HM5JICrp6CVYIwalTxykszAPAYglg5sy5RET4ppmz2+3l6/WF\nXC7XWp3EJoQw756hmC09L2XfCLyql4+KPudEnTZ/NzYwmmeGryLC4v/+hQAdTbk0V2wGBCh6olOW\nEBiWAUDZ2eOc2PHJlUIePePmPkhSahaOPW/gKTsJgC4sHn3mk9hPavddqB5ai7bgaikn/uGVhI6f\n4K9L8ytSBCUSiURyU9Ha4eSltblUNWiVoiPTY3hsQRYGffcb+4UQfH25iUO1mgAEG/Q8nJFEfODV\nS7BCCE6c+IZz57Q5tIGBQcycOZewMN/IkMvpYdu6M9RUtgKQkBzG7XcNwWT27+PYo3p4v/AzchsK\nAEgMimf18JWEmf3fugagveE4LZe/BkDRGYlJvQdLSCoAF3IPcWrvFwDoDUYmzH+U+ORU7Dv/hrdS\nux4lsi+61Edx5GrfAdXrorXwK9wdNSQ8/iQhI2/xw1X1DqQISiQSieSmobnNwR/X5FLXrFWDjs2K\n45G5g9D3UN2pCsGWiga+rdfEK8xo4JHMJKItVy/BqqrKt98eoqSkGIDg4BBmzpxLSIhvmjk7HW62\nrC2gvkbLYib3j+C2xVkYjXqfnP96cXndvHvmIwqbzgHQJySJ1cMfJdjo/9Y1AK21h2it0YpWFL2Z\n2NRlmIO1yuWi47vIP7wZAKPJwqSFjxEdm4h925/w1mrj+ZTYNH
```

Comment:
Please remove Warning line.

Reply:
I solved the warning for local installs.